Document/0x06f-Testing-Local-Authentication.md

```diff
@@ -199,7 +199,7 @@ It is important to remember that the LocalAuthentication framework is an event-b
 - Verify that access control flags are set for the keychain item which ensure that the keychain item its data can only be unlocked by means of authenticating the user. This can be with one of the following flags:
```

Comment:
```suggestion
- Verify that access control flags are set for the keychain item which ensure that the data of the keychain item can only be unlocked by means of authenticating the user. This can be done with one of the following flags:
```